src/components/search-results.js

```diff
@@ -384,8 +387,11 @@ class SearchResults extends SkeletonMixin(FetchMixin(LocalizeLitMixin(RouteLocat
 			composed: true
 		}));
 		this._showLoadingOverlay = true;
-		const discoverList = this.shadowRoot.querySelector('#discover-search-results-list');
-		discoverList.Reset();
+		const discoverList = this.shadowRoot?.querySelector('#discover-search-results-list');
+
+		if (discoverList) {
+			discoverList.Reset();
```

Comment:
```suggestion
			discoverList?.Reset();
```
Then the if wrapper is not needed.